### DISCUSSION TITLE
2006/11/01/%E3%80%8A%E7%BB%8F%E6%B5%8E%E5%AD%A6%E5%8E%9F%E7%90%86%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#126] [Announcements]
# 2006/11/01/%E3%80%8A%E7%BB%8F%E6%B5%8E%E5%AD%A6%E5%8E%9F%E7%90%86%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

这本书的英文名是：Principles of Economics，作者：N. Gregory Mankiw。依我看来，这是一本每个人都应当读一读的好书，若你对经济学知之甚少，则尤其如此。我看的这个版本是它的第三版中文版，由梁小民翻译，分上下两册。现在最新的英文版已经是第五版了。我现在只读完了它的上册，即前十七章。我打算先暂停一段时间，看看其他的书，然后再继续学习它的下册。

https://blog.qizhen.xyz/2006/11/01/%E3%80%8A%E7%BB%8F%E6%B5%8E%E5%AD%A6%E5%8E%9F%E7%90%86%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

春辉
2009年07月1日 16:54 编辑
善于客观评价自己的现状，善于归纳，善于学习理论，善于理论联系实际，善于更好的理解生活的本质。有不同种选择，学一下next second,想象一下预见一下，没有坏处

回复
高阳
2007年12月10日 05:24 编辑
呵呵，我是突然心血来潮搜索这本书进来的，有幸拜读你的读后感。挺有味的。不过还没有看过这本书，最近有想法要多补充点经济学的专业知识，头脑里就蹦出来这个书名，打算找来看看。

回复
传杰
2006年11月29日 14:02 编辑
的确是本好书，读了很多次了

回复
Qizhen
2006年11月12日 20:31 编辑
早就工作啦，

回复
Suzanne
2006年11月12日 13:38 编辑
请问师兄你现在在什么大学？
我想认识你一下
我的msn:dormouse117@hotmail.com

回复
x
2006年11月7日 17:20 编辑
请问什么是LabVIEW ？
为什么会邀请我参加您的朋友圈呢？

回复
拉布拉多
2006年11月7日 09:46 编辑
写的很有道理的文章，非常喜欢。得到和失去有时候必须衡量。

回复
竹贤
2006年10月31日 14:01 编辑
把权衡取舍作为经济学十大原理之首,说明了它是最基本和最重要的，但要正确、准确地处理好取舍实在是不容易啊。购物中的取舍是最简单最容易判断权衡的，因为相对客观也容易比较。事实上人生的每一步似乎都面临着取舍：也许在你未成年之前由他人为你取舍；在你长大后，由你自己来取舍（其实在很多情况下，取舍还是有限制条件的）。
在权衡取舍的时候，人人都会趋利避害，或是两利相权取其重。道理很明确，但很多时候利弊常常交融在一起，让你一时难以明辨；或者利在眼前，弊在以后，让人难以判断；更有甚者，也许由于形势的变化，今天的利到了明天就成了弊……
所以，仅仅知道了经济学的原理还远远不够，还要有健全的心理和心态，有正确的判断力和决策能力，又承受挫折的能力，要学会客观地分析形势，看清楚形势的走向。
努力吧，小伙子。

回复
竹贤
2006年10月27日 22:21 编辑
这类书当然值得一读。建议你在读此类书时，随读随记笔记随写心得.当你读完一册，把这些东西整理整理，提炼提炼，综合综合，就是一篇好文章。对你来说，也许更有帮助。

回复
